cpp/examples/parquet/low-level-api/reader-writer.cc

```diff
@@ -205,6 +205,8 @@ int main(int argc, char** argv) {
       int i;
       std::shared_ptr<parquet::ColumnReader> column_reader;
 
+      (void)rows_read; // prevent warning in release build
```

Comment:
We have a `ARROW_UNUSED` macro for this.

Reply:
Thanks!  I'll add a commit to use it.